symfony/console/3.3/bin/console

```diff
@@ -15,15 +14,11 @@ if (!class_exists(Application::class)) {
     throw new \RuntimeException('You need to add "symfony/framework-bundle" as a Composer dependency.');
 }
 
-if (!isset($_SERVER['APP_ENV'])) {
-    if (!class_exists(Dotenv::class)) {
-        throw new \RuntimeException('APP_ENV environment variable is not defined. You need to define environment variables for configuration or add "symfony/dotenv" as a Composer dependency to load variables from a .env file.');
-    }
-    (new Dotenv())->load(__DIR__.'/../.env');
-}
-
 $input = new ArgvInput();
 $env = $input->getParameterOption(['--env', '-e'], $_SERVER['APP_ENV'] ?? 'dev', true);
+
+Kernel::bootstrapEnvironment($env);
```

Comment:
I see an issue here: as soon as you use `--env`, the .env file won't be loaded anymore, meaning the flag won't be usable anymore locally.

Reply:
even when you don't use it btw, due to the default to `dev`

Reply:
@stof not sure what you mean, the env will always be loaded if there is a .env file and DotEnv is registered. It will also load both .env and `.env.{ENV}`. The default Server APP_ENV that is for, say, production. Whereas the default would then be `prod` for the cases without the option defined and then bootstrap would not happen.

Reply:
@stof i've changed so it loads `.env` and after that load whatever `.env.$serverAppEnv` was set there